### PR TITLE
feat: Application metrics data test ids

### PR DESCRIPTION
**Pull request by Ash-exp**
# Description

This PR contains the require Data test ids for the UI automation of [App Metrics](https://github.com/deepak-devtron/dashboard-automation/pull/157)

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

- [x] UI automation

# Checklist:

* [x] The title of the PR states what changed and the related issues number (used for the release note).
* [ ] Does this PR require documentation updates?
* [ ] I've updated documentation as required by this PR.
* [x] I have performed a self-review of my own code
* [ ] I have commented my code, particularly in hard-to-understand areas

### CODE REVIEW

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx [hunk 1]
```diff
@@ -330,7 +337,7 @@ const InstallationStatus = ({
             {(installationStatus === ModuleStatus.INSTALLING || installationStatus === ModuleStatus.UPGRADING) && (
                 <>
                     <Progressing size={24} />
-                    <div className="mt-12 dc__loading-dots">
+                    <div data-test-id="module-status-progressing" className="mt-12 dc__loading-dots">
```
Comment (milindk8) [inline src/components/v2/devtronStackManager/DevtronStackManager.component.tsx:340]: data-testid not data-test-id

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx [hunk 2]
```diff
@@ -340,13 +347,19 @@ const InstallationStatus = ({
                 (installationStatus === ModuleStatus.HEALTHY && !latestVersionAvailable)) && (
                 <>
                     {isUpgradeView ? (
-                        <div className="module-details__upgrade-success flex column">
+                        <div
+                            data-test-id="module-status-updated"
```
Comment (milindk8) [inline src/components/v2/devtronStackManager/DevtronStackManager.component.tsx:351]: same here

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx [hunk 3]
```diff
@@ -340,13 +347,19 @@ const InstallationStatus = ({
...
+                            className="module-details__upgrade-success flex column"
+                        >
                             <img src={LatestVersionCelebration} />
                             <UpToDateIcon className="icon-dim-40" />
                             <span className="mt-12">You're using the latest version of Devtron.</span>
                         </div>
                     ) : (
-                        <div className="module-details__installtion-success flex left">
+                        <div
+                            data-test-id="module-status-installed"
```
Comment (milindk8) [inline src/components/v2/devtronStackManager/DevtronStackManager.component.tsx:360]: here too

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx [hunk 4]
```diff
@@ -356,7 +369,7 @@ const InstallationStatus = ({
                 installationStatus === ModuleStatus.UPGRADE_FAILED ||
                 installationStatus === ModuleStatus.TIMEOUT ||
                 installationStatus === ModuleStatus.UNKNOWN) && (
-                <div className="module-details__installtion-failed flex left">
+                <div data-test-id="module-status-failed" className="module-details__installtion-failed flex left">
```
Comment (milindk8) [inline src/components/v2/devtronStackManager/DevtronStackManager.component.tsx:372]: here as well

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx (hunk 3 above)
Comment (Ash-exp) [reply src/components/v2/devtronStackManager/DevtronStackManager.component.tsx]: updated

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx (hunk 4 above)
Comment (Ash-exp) [reply src/components/v2/devtronStackManager/DevtronStackManager.component.tsx]: updated

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx (hunk 2 above)
Comment (Ash-exp) [reply src/components/v2/devtronStackManager/DevtronStackManager.component.tsx]: updated

File: src/components/v2/devtronStackManager/DevtronStackManager.component.tsx (hunk 1 above)
Comment (Ash-exp) [reply src/components/v2/devtronStackManager/DevtronStackManager.component.tsx]: updated

**Review by vivek-devtron — APPROVED** [review]

**Pull request merged**